### PR TITLE
add: MirrorMaker troubleshooting section and first guide

### PR DESCRIPTION
**Pull request by jclarysse**
## Describe your changes

Create a new menu section dedicated to troubleshooting MirrorMaker, along with first guide "Why are some topics or partitions not replicated"

[DOC-1212]

Page preview - https://jclarysse-add-mirrormaker-tr.aiven-docs.pages.dev/docs/products/kafka/kafka-mirrormaker/troubleshooting/topic-not-replicated

## Checklist

- [x] The first paragraph of the page is on one line.
- [x] The other lines have a line break at 90 characters.
- [x] I checked the output.
- [x] I applied the [style guide](styleguide.md).
- [x] My links start with `/docs/`.

[DOC-1212]: https://aiven.atlassian.net/browse/DOC-1212?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/products/kafka/kafka-mirrormaker/troubleshooting/topic-not-replicated.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+and has assessed an issue with the replication of some topics or partitions.
+
+:::note
+There are different ways of performing this issue assesment, including basic monitoring
+and [offset sync status analysis](/docs/products/kafka/kafka-mirrormaker/howto/log-analysis-offset-sync-tool).
+:::
+
+## RecordTooLargeException is observed in the logs
+
+This can be fixed by increasing the value of broker config message_max_bytes at target, as well as integration config [producer_max_request_size](https://registry.terraform.io/providers/aiven/aiven/latest/docs/resources/service_integration#producer_max_request_size-1). Following to this change, it is necessary that the workers restart their task.
```
Comment (jjaakola-aiven) [inline docs/products/kafka/kafka-mirrormaker/troubleshooting/topic-not-replicated.md:18]: This is visible in the logs for customer too, I would emphasize self-service aspect for correcting this.

To add more detail here, there are two possibilities for this error to pop up.
1. **Kafka broker** rejects the record if it is larger than the topic configuration.
   * Correct the configuration for topic in the Kafka cluster. 
1. **Mirrormaker producer** rejects
   * Correct the configuration in Mirrormaker integration configuration to target Kafka.

File: docs/products/kafka/kafka-mirrormaker/troubleshooting/topic-not-replicated.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+## Configuration issue
+
+The following two worker parameters may significantly impact how topic partitions are replicated:
+
+- [kafka_mirrormaker.offset_lag_max](/docs/products/kafka/kafka-mirrormaker/reference/advanced-params#kafka_mirrormaker_offset_lag_max) (100 by default) defines how out-of-sync a remote partition can be before it is re-synced. Given that this parameter is global to all replication topics, you need to find a compromise between:
```
Comment (jjaakola-aiven) [inline docs/products/kafka/kafka-mirrormaker/troubleshooting/topic-not-replicated.md:24]: The offset sync is emitted also periodically from Kafka 3.7 onwards, currently the MM2 version run in Aiven: https://issues.apache.org/jira/browse/KAFKA-15906

File: docs/products/kafka/kafka-mirrormaker/troubleshooting/topic-not-replicated.md (hunk 1 above)
Comment (jclarysse) [reply docs/products/kafka/kafka-mirrormaker/troubleshooting/topic-not-replicated.md]: Added, Thanks!

File: docs/products/kafka/kafka-mirrormaker/troubleshooting/topic-not-replicated.md (hunk 2 above)
Comment (jclarysse) [reply docs/products/kafka/kafka-mirrormaker/troubleshooting/topic-not-replicated.md]: Added, Thanks!

**Review by harshini-rangaswamy — APPROVED** [review]
Thank you for adding this content, @jclarysse. I’ve made a few adjustments to align it with our style guidelines and to improve the overall flow for clarity and readability.

**Pull request merged**